### ISSUE TITLE
BUG: import static with wildcard cannot be compiled.

### ISSUE THREAD

**Issue by danglotb** [INRIA/spoon#2780]
Hi all.

It seems that Spoon is not able anymore to compile java classes with a static import using a wildcard.

I created a JUnit test to reproduce the bug, see this [branch](https://github.com/danglotb/spoon/tree/bug-import-static-wildcard) or use the following lines:

```sh
git remote add danglotb https://github.com/danglotb/spoon
git fetch danglotb bug-import-static-wildcard
git checkout bug-import-static-wildcard
```

The corresponding failing test is `spoon.test.compilation.CompilationTest#compileTestWithImportStaticWildcard()`.

Bug for assertfixer

**Comment by plcarmel** [comment]
I don't know if it is related but I loose the wildcards on static imports when pretty printing the model.

I observed the behavior on  7.2.0 and 7.3.0-SNAPSHOT (commit 1b70c10660002d6a193f33d849a45958a92183aa).

**Comment by plcarmel** [comment]
I did a git bisect and it seems my problem starts occurring here:

```
a29633c140f6349b51d072329525ce4c0d95f28b is the first bad commit
commit a29633c140f6349b51d072329525ce4c0d95f28b
Author: Pavel Vojtechovsky <pvojtechovsky@users.noreply.github.com>
Date:   Mon Nov 5 21:02:09 2018 +0100

    feat: new CtTypeMemberWildcardImportReference for `import package.Type.*` (#2731)

:040000 040000 cf0e2d51aced573f042e543fcd6b10fceafd6cdd 4f2485d948fa9e10a43fcb732d69f343b3853b46 M    src
```

**Comment by plcarmel** [comment]
@danglotb , I confirm that the aforementioned commit is also the one that make your test fail. Our two problems have the same cause.

**Comment by pvojtechovsky** [comment]
please try it with code of this PR #2683. It is going to be a new implementation of import handlers.

**Comment by plcarmel** [comment]
I confirm that I don't have the problem with PR #2683

**Comment by pvojtechovsky** [comment]
The #2683 is waiting in queue of PRs for review. It will need some time, because the changes was not primitive, so review time is needed. And may be I have to refactor #2683 before it can be merged.

**Comment by monperrus** [comment]
Closed by #2865 

Benjamin, there seems to another error in AssertFixer, could you have a look?
